components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -71,6 +74,33 @@ func (hc *Client) ReleaseStatus(rname string) (string, error) {
 	return strings.Replace(statusStr, `\n`, "\n", -1), nil
 }
 
+//IsReleaseDeletable returns true for release that can be deleted
+func (hc *Client) IsReleaseDeletable(rname string) (bool, error) {
+	isDeletable := false
+	maxAttempts := 3
+
+	err := retry.Do(
+		func() error {
+			statusRes, err := hc.helm.ReleaseStatus(rname)
+			if err != nil {
```

Comment:
probably we'll need to check here for not found error. `helm status xxx` reports `Error: getting deployed release "xxx": release: "xxx" not found` for nonexisting releases (which is the case when error is returned on prepare-release phase in tiller)